tests/test_emotes.py

```diff
@@ -0,0 +1,6 @@
+from configparser import ConfigParser
+from server.emotes import Emotes
+
+
+def test_create_config_parser_return_type():
+    assert Emotes._create_config_parser() == ConfigParser()
```

Comment:
Huh I appreciate the enthusiasm but this is not a very good test

1. You should only test public-facing methods and functionality. Private internal methods are seldom tested because they are considered implementation details that are subject to change.
2. `_create_config_parser()` creates a parameterized `ConfigParser` so I am not sure why this is equal to a default `ConfigParser()`. If anything you would test the type, but this would be redundant because the linter should already be doing type checking.